Connected System Template (CST) Examples/Data Entry (Dynamic Data Structures)/src/main/java/com/dataentry/forms/templates/DynamicDataStructureIntegrationTemplate.java

```diff
@@ -190,11 +191,11 @@ private TextPropertyDescriptor createFormDropdown(JsonNode dataTypes) {
 
   private PropertyDescriptorBuilder parsePropertyDescriptorType(String type) {
     if (SUPPORTED_PROPERTY_TYPE.TEXT.name().equals(type)) {
-      return DomainSpecificLanguage.textProperty();
+      return TextPropertyDescriptor.builder();
```

Comment:
Use the methods on "ConfigurableTemplate"

Reply:
is that the name of the class? I can't find it 😕